### DISCUSSION TITLE
[Feature] Show location of file in Review Duplicates

### DISCUSSION THREAD

**Discussion by elsingaa** [immich-app/immich#12851] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be very useful to see the location of the duplicate files when using Review Duplicates.

After years of taking photos with different phones, backing up to laptop, and backing that up to a NAS, I have a lot of duplicates (for some files more than 8. I know, that's a "me-problem"). 

The workflow is now:
- clicking the magnifying glass on the image,
- click the (I)nfo button on top of the screenif not shown already
- click the (I)nfo button after the file name
- click right/left for the other files
- press escape, remembering how often you went left or right, otherwise loosing which file was the right one
- Keep the correct file, Trash the other ones

When the file location is shown in the 'select screen' the flow is:
- Keep the correct file, Trash the other ones

That would save a lot of time

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by yu-jingrui** [comment]
This is the first thing that I want after duplicate detection finally working on my server.

**Comment by bo0tzz** [comment]
#22632
